index.js

```diff
@@ -37,7 +38,9 @@ module.exports = {
     global.EmberENV = EmberENV;
 
     var Compiler = require(templateCompilerPath);
-    var PrecompileInlineHTMLBarsPlugin = HTMLBarsInlinePrecompilePlugin(Compiler.precompile); // jshint ignore:line
+    var templateCompilerFullPath = require.resolve(templateCompilerPath);
+    var templateCompilerCacheKey = fs.readFileSync(templateCompilerFullPath, { encoding: 'utf-8' });
+    var PrecompileInlineHTMLBarsPlugin = HTMLBarsInlinePrecompilePlugin(Compiler.precompile, templateCompilerCacheKey); // jshint ignore:line
```

Comment:
> RE: checksum vs content, it doesn't much matter to me, I'm happy to go either way.

lets leave it as is, it gets checksumed in the plugin serialization phase anyways correct?